ingestion_server/ingestion_server/es_helpers.py

```diff
@@ -55,19 +59,18 @@ def _elasticsearch_connect() -> Elasticsearch:
 
     timeout = 12  # hours
 
-    log.info(f"Connecting to {elasticsearch_url}:{elasticsearch_port} with AWS auth")
+    log.info(f"Connecting to {es_endpoint} with AWS auth")
     auth = AWSRequestsAuth(
         aws_access_key=aws_access_key_id,
         aws_secret_access_key=aws_secret_access_key,
-        aws_host=elasticsearch_url,
+        aws_host=es_url,
         aws_region=aws_region,
         aws_service="es",
     )
     auth.encode = lambda x: bytes(x.encode("utf-8"))
     es = Elasticsearch(
-        host=elasticsearch_url,
-        port=elasticsearch_port,
-        connection_class=RequestsHttpConnection,
+        es_endpoint,
+        node_class=RequestsHttpNode,
         http_auth=auth,
```

Comment:
`http_auth` from line 74 is [deprecated](https://www.elastic.co/guide/en/elasticsearch/client/python-api/master/migration.html#migration-options), and should be replaced with `basic_auth`.

Reply:
Thanks for catching that!

Reply:
BTW, do you know if we still need this authentication stuff? Our clusters are not accessible on the public internet and we don't need any auth to make connections via our jumpbox, for example :thinking: This is old code, and I'm not sure how to determine whether it's necessary. Something to keep an eye on though in case these parameters keep changing or get more complex.

Reply:
I've removed it in https://github.com/WordPress/openverse/pull/3029/commits/f138fec3e865d5e1f4959c436c8f435a56259286 as per the other conversation about the parameter.

Reply:
> BTW, do you know if we still need this authentication stuff? 

I really don't know, and your explanations sound reasonable. Let's try :)